tox.ini

```diff
@@ -32,7 +32,7 @@ commands =
 deps =
      -r{toxinidir}/requirements_test.txt
 commands =
-    python script/gen_requirements_all.py validate
+    python -m script.gen_requirements_all validate
```

Comment:
Please make a separate PR for this.

Reply:
Woops. Didn't mean to include this. I'll move it out.